src/kibana/plugins/vis_types/vislib/editors/area.html

```diff
@@ -6,3 +6,9 @@
   </label>
 </div>
 <vislib-basic-options></vislib-basic-options>
+<div class="vis-option-item">
+  <label>
+    <input type="checkbox" value="{{defaultYExtents}}" ng-model="vis.params.defaultYExtents" name="defaultYExtents" ng-checked="vis.params.defaultYExtents">
+    Set y axis to data extents
```

Comment:
We probably need new verbage here. Perhaps having a conversation with @w33ble or @spenceralger.